deploy/clowdapp.yaml

```diff
@@ -62,8 +62,8 @@ objects:
           value: ${CLOWDER_ENABLED}
         - name: INGRESS_MINIOENDPOINT
           value: ${INGRESS_MINIOENDPOINT}
-        - name: INGRESS_BLACK_LISTED_ORGIDS
-          value: ${INGRESS_BLACK_LISTED_ORGIDS}
+        - name: INGRESS_DENY_LISTED_ORGIDS
+          value: ${INGRESS_DENY_LISTED_ORGIDS}
```

Comment:
Thanks for the PR @SteveHNH.  We'll need a corresponding MR for this as well.

Wonder if we create that now so we don't forget.  We could either duplicate this var for a short time or just make sure we are mindful when pushing out this new version that this change is needed.

Reply:
oh yeah! I figured we had cleaned those up, but sure enough we still have one listed. I can do an MR to just duplicate it so we have both. Then we can merge this one when you're ready

Reply:
https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/80026